deps/libid3tag/libid3tag.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
I suggest to get rid of this whole file and just build libid3tag in the main YAML manifest.